typescript/packages/jumble/src/components/CharmRunner.tsx

```diff
@@ -80,7 +80,7 @@ export function CharmRenderer({ charm, className = "" }: CharmRendererProps) {
   const containerRef = useRef<HTMLDivElement>(null);
   const [runtimeError, setRuntimeError] = React.useState<Error | null>(null);
   const [isFixing, setIsFixing] = React.useState(false);
-  const { charmManager, currentReplica } = useCharmManager();
+  const { spaceManager: charmManager, currentSpaceURI: currentReplica } = useSpaceManager();
```

Comment:
let's rename the use site instead of aliasing the variable?

Reply:
done everywhere (missed that earlier in the auto-rename, sorry!)